content/Box1: experimental design

```diff
@@ -0,0 +1,23 @@
+# Designing experiments for applied machine learning
```

Comment:
I am going to propose a little bit of a reorganization that I think will help shorten things up.

#### First paragraph

* Machine learning models are trained on a training set <- define training set in this same sentence
* Generalization is important because xyz...
* To understand how well a model generalizes, we need a test set with xyz features (e.g., similar patterns, class balance, what have you) 
* A test set is only seen by the model once training is complete, e.g., held-out
* People sometimes split the same dataset 75/25. 
* You _really need a training set_

#### Second paragraph

* There's also something called cross-validation. Here are some benefits (e.g., model is tested on all of the data)
* There are many kinds, let's focus on k-fold cross-validation (I think this one is easiest to explain)
* Oh by the way, ML models have parameters let's talk about tuning
* (I think it might be overkill to talk about things like data leakage, nested CV, etc. here. Not that you do that currently. More that I am typing "out loud.")

A box needs _minimal wrap-up_ in my opinion. Instead, we could try weaving the information you have in the last paragraph (e.g., standardizing features) and put it right where folks "need it" in the other two paragraphs.